src/Web/Masa.Scheduler.Web.Admin/Pages/Teams/Components/JobModal.razor

```diff
@@ -213,7 +213,7 @@
                                         </MRow>
                                     </AutoHeightContent>
                                     <FooterContent>
-                                        <div class="d-flex mt-9">
+                                                <div class="d-flex mt-9" style="align-items: center;">
```

Comment:
align-center    Please try to use built-in css instead of style

Reply:
this is flex layout. align-center not effect. must use align-items

Reply:
![image](https://user-images.githubusercontent.com/38368335/222346866-b59d336d-9a10-492f-9440-49bd2c08cee0.png)

Reply:
Align-center is the built-in css of masa blazer. Its effect is to align items: center   @MapleWithoutWords

Reply:
> Align-center is the built-in css of masa blazer. Its effect is to align items: center @MapleWithoutWords

ok,already changed.